composer.json

```
@@ -29,6 +29,7 @@
     },
     "require": {
         "php": ">=7.1",
+        "ext-json": "^1.7",
```

```suggestion
        "ext-json": "*",
```
since we don't really depend on a specific version